Dockerfile

```diff
@@ -3,14 +3,17 @@
 FROM python:3.6-stretch
 
 WORKDIR /app
-COPY . /app
+COPY Pipfile /app
```

Comment:
In this case, the `Pipfile` does not need to be committed separately. The `Pipfile.lock` is used for installing dependencies in the `pipenv install` step.

Reply:
So removing that line make the pipenv install command on line 13 fail with this error: 

ERROR: Pipfile not found! Aborting! Please ensure that the file exists and is located in your project root directory.

So copying the Pipfile over with the lock file is required.